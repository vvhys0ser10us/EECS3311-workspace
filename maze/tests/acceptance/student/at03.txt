move(E)
new_game(easy)
move(E)
move(E)
move(S)
move(S)
move(E)
move(S)
move(S)
move(S)
move(S)
move(N)
move(N)
move(N)
move(N)
move(E)
move(E)
move(S)
move(W)
move(S)
move(E)
move(S)
move(S)
move(W)
move(W)
move(E)
new_game(easy)
move(E)
move(E)