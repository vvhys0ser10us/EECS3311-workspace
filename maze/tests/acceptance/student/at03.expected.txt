  State: 0 -> ok
->move(E)
  State: 1 -> Error! Not in a game.
->new_game(easy)
  State: 2 -> ok
  +-*-+---+---+---+---+---+---+
  | X                         |
  +   +   +   +---+---+   +   +
  |       |   |       |       |
  +   +---+   +---+   +---+   +
  |   |                   |   |
  +   +   +   +   +---+   +   +
  |   |       |   |       |   |
  +   +   +---+   +   +   +   +
  |       |   |   |           |
  +---+   +   +   +---+   +---+
  |   |   |       |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 3 -> ok
  +-*-+---+---+---+---+---+---+
  | * * X                     |
  +   +   +   +---+---+   +   +
  |       |   |       |       |
  +   +---+   +---+   +---+   +
  |   |                   |   |
  +   +   +   +   +---+   +   +
  |   |       |   |       |   |
  +   +   +---+   +   +   +   +
  |       |   |   |           |
  +---+   +   +   +---+   +---+
  |   |   |       |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 4 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * X                 |
  +   +   +   +---+---+   +   +
  |       |   |       |       |
  +   +---+   +---+   +---+   +
  |   |                   |   |
  +   +   +   +   +---+   +   +
  |   |       |   |       |   |
  +   +   +---+   +   +   +   +
  |       |   |   |           |
  +---+   +   +   +---+   +---+
  |   |   |       |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 5 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | X |       |       |
  +   +---+   +---+   +---+   +
  |   |                   |   |
  +   +   +   +   +---+   +   +
  |   |       |   |       |   |
  +   +   +---+   +   +   +   +
  |       |   |   |           |
  +---+   +   +   +---+   +---+
  |   |   |       |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 6 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     X             |   |
  +   +   +   +   +---+   +   +
  |   |       |   |       |   |
  +   +   +---+   +   +   +   +
  |       |   |   |           |
  +---+   +   +   +---+   +---+
  |   |   |       |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 7 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * X         |   |
  +   +   +   +   +---+   +   +
  |   |       |   |       |   |
  +   +   +---+   +   +   +   +
  |       |   |   |           |
  +---+   +   +   +---+   +---+
  |   |   |       |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 8 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * *         |   |
  +   +   +   + * +---+   +   +
  |   |       | X |       |   |
  +   +   +---+   +   +   +   +
  |       |   |   |           |
  +---+   +   +   +---+   +---+
  |   |   |       |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 9 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * *         |   |
  +   +   +   + * +---+   +   +
  |   |       | * |       |   |
  +   +   +---+ * +   +   +   +
  |       |   | X |           |
  +---+   +   +   +---+   +---+
  |   |   |       |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 10 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * *         |   |
  +   +   +   + * +---+   +   +
  |   |       | * |       |   |
  +   +   +---+ * +   +   +   +
  |       |   | * |           |
  +---+   +   + * +---+   +---+
  |   |   |     X |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 11 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * *         |   |
  +   +   +   + * +---+   +   +
  |   |       | * |       |   |
  +   +   +---+ * +   +   +   +
  |       |   | * |           |
  +---+   +   + * +---+   +---+
  |   |   |     * |       |   |
  +   +   +   + * +---+   +   +
  |       |     X             |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(N)
  State: 12 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * *         |   |
  +   +   +   + * +---+   +   +
  |   |       | * |       |   |
  +   +   +---+ * +   +   +   +
  |       |   | * |           |
  +---+   +   + * +---+   +---+
  |   |   |     X |       |   |
  +   +   +   + * +---+   +   +
  |       |     *             |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(N)
  State: 13 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * *         |   |
  +   +   +   + * +---+   +   +
  |   |       | * |       |   |
  +   +   +---+ * +   +   +   +
  |       |   | X |           |
  +---+   +   + * +---+   +---+
  |   |   |     * |       |   |
  +   +   +   + * +---+   +   +
  |       |     *             |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(N)
  State: 14 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * *         |   |
  +   +   +   + * +---+   +   +
  |   |       | X |       |   |
  +   +   +---+ * +   +   +   +
  |       |   | * |           |
  +---+   +   + * +---+   +---+
  |   |   |     * |       |   |
  +   +   +   + * +---+   +   +
  |       |     *             |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(N)
  State: 15 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * X         |   |
  +   +   +   + * +---+   +   +
  |   |       | * |       |   |
  +   +   +---+ * +   +   +   +
  |       |   | * |           |
  +---+   +   + * +---+   +---+
  |   |   |     * |       |   |
  +   +   +   + * +---+   +   +
  |       |     *             |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 16 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * X     |   |
  +   +   +   + * +---+   +   +
  |   |       | * |       |   |
  +   +   +---+ * +   +   +   +
  |       |   | * |           |
  +---+   +   + * +---+   +---+
  |   |   |     * |       |   |
  +   +   +   + * +---+   +   +
  |       |     *             |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 17 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * * * X |   |
  +   +   +   + * +---+   +   +
  |   |       | * |       |   |
  +   +   +---+ * +   +   +   +
  |       |   | * |           |
  +---+   +   + * +---+   +---+
  |   |   |     * |       |   |
  +   +   +   + * +---+   +   +
  |       |     *             |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 18 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * * * * |   |
  +   +   +   + * +---+ * +   +
  |   |       | * |     X |   |
  +   +   +---+ * +   +   +   +
  |       |   | * |           |
  +---+   +   + * +---+   +---+
  |   |   |     * |       |   |
  +   +   +   + * +---+   +   +
  |       |     *             |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(W)
  State: 19 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * * * * |   |
  +   +   +   + * +---+ * +   +
  |   |       | * | X * * |   |
  +   +   +---+ * +   +   +   +
  |       |   | * |           |
  +---+   +   + * +---+   +---+
  |   |   |     * |       |   |
  +   +   +   + * +---+   +   +
  |       |     *             |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 20 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * * * * |   |
  +   +   +   + * +---+ * +   +
  |   |       | * | * * * |   |
  +   +   +---+ * + * +   +   +
  |       |   | * | X         |
  +---+   +   + * +---+   +---+
  |   |   |     * |       |   |
  +   +   +   + * +---+   +   +
  |       |     *             |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 21 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * * * * |   |
  +   +   +   + * +---+ * +   +
  |   |       | * | * * * |   |
  +   +   +---+ * + * +   +   +
  |       |   | * | * * X     |
  +---+   +   + * +---+   +---+
  |   |   |     * |       |   |
  +   +   +   + * +---+   +   +
  |       |     *             |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 22 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * * * * |   |
  +   +   +   + * +---+ * +   +
  |   |       | * | * * * |   |
  +   +   +---+ * + * +   +   +
  |       |   | * | * * *     |
  +---+   +   + * +---+ * +---+
  |   |   |     * |     X |   |
  +   +   +   + * +---+   +   +
  |       |     *             |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 23 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * * * * |   |
  +   +   +   + * +---+ * +   +
  |   |       | * | * * * |   |
  +   +   +---+ * + * +   +   +
  |       |   | * | * * *     |
  +---+   +   + * +---+ * +---+
  |   |   |     * |     * |   |
  +   +   +   + * +---+ * +   +
  |       |     *       X     |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(W)
  State: 24 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * * * * |   |
  +   +   +   + * +---+ * +   +
  |   |       | * | * * * |   |
  +   +   +---+ * + * +   +   +
  |       |   | * | * * *     |
  +---+   +   + * +---+ * +---+
  |   |   |     * |     * |   |
  +   +   +   + * +---+ * +   +
  |       |     *   X * *     |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(W)
  State: 25 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * * * * |   |
  +   +   +   + * +---+ * +   +
  |   |       | * | * * * |   |
  +   +   +---+ * + * +   +   +
  |       |   | * | * * *     |
  +---+   +   + * +---+ * +---+
  |   |   |     * |     * |   |
  +   +   +   + * +---+ * +   +
  |       |     X * * * *     |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 26 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * * * * |   |
  +   +   +   + * +---+ * +   +
  |   |       | * | * * * |   |
  +   +   +---+ * + * +   +   +
  |       |   | * | * * *     |
  +---+   +   + * +---+ * +---+
  |   |   |     * |     * |   |
  +   +   +   + * +---+ * +   +
  |       |     * * X * *     |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->new_game(easy)
  State: 27 -> Error! In game already.
  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 28 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * * * * |   |
  +   +   +   + * +---+ * +   +
  |   |       | * | * * * |   |
  +   +   +---+ * + * +   +   +
  |       |   | * | * * *     |
  +---+   +   + * +---+ * +---+
  |   |   |     * |     * |   |
  +   +   +   + * +---+ * +   +
  |       |     * * * * X     |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 29 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * *                 |
  +   +   + * +---+---+   +   +
  |       | * |       |       |
  +   +---+ * +---+   +---+   +
  |   |     * * * * * * * |   |
  +   +   +   + * +---+ * +   +
  |   |       | * | * * * |   |
  +   +   +---+ * + * +   +   +
  |       |   | * | * * *     |
  +---+   +   + * +---+ * +---+
  |   |   |     * |     * |   |
  +   +   +   + * +---+ * +   +
  |       |     * * * * * * * |
  +---+---+---+---+---+---+-X-+

  Congratulations! You escaped the maze!
  Game Number: 1
  Score: 1 / 1

